docs/admin/overview.md

```diff
@@ -0,0 +1,22 @@
+---
+id: overview
+title: "Administration"
+sidebar_label: "Administration"
+---
+
+The admin section of Scoreboard allows admin users to manage badges, teams, and users.
+
+## [Badges](./badges)
+
+- [Create new badges](./create-badge)
+- [Edit badges](./edit-badge)
+
+## [Teams](./teams)
+
+- [Create new teams](./create-team)
+- [Add campaigns to teams](./add-team-campaigns)
+- [Add users to teams](./add-team-users)
+
+## [Users](./users)
+
+- [Make users admins](./make-users-admins)
```

Comment:
This and the sidebar should have the same ordering of pages, so they should both be either `Badges, Teams, Users` or `Users, Teams, Badges`.

I also don't think it's necessary to list every subpage since the same info is available on the sidebar